packages/fixie/src/voice.ts

```diff
@@ -133,6 +148,7 @@ export class VoiceSession {
       this.sendData(obj);
     }, 5000);
     this.audioStarted = true;
+    this.onAudioStart?.();
```

Comment:
I assumed this would be useful here because if the user declined mic access presumably (I am not sure?) the localAudioTrack would not be populated and then we would not fire this event. The idea was to only fire the event once mic access had been granted and obtained by the app. WDYT?

Reply:
we'll block in this method until we get an answer from the user. If the user says no we'll get a permission-denied `DOMException` which the app can catch when calling `startAudio`.

Reply:
IOW I still tend to think the event is not valuable here since it's just tracking whether a single method completed.

Reply:
OK, removed it!